app/views/application/_header.html.erb

```diff
@@ -16,6 +16,11 @@
       </div>
 
       <ul class="nav navbar-nav navbar-right">
+        <% if @current_user %>
+          <%= header_nav_item t('site-header.sign-out'), auth_sign_out_path %>
+        <% else %>
+          <%= header_nav_item t('site-header.sign-in'), auth_sign_in_path %>
+        <% end %>
```

Comment:
@jits I need your help on this. I cannot for the life of me get this to work correctly on the landing page. On the Subscription page it shows the correct link and I think it may be due to how Turbolinks is caching things, but I am not sure

Reply:
Sure thing, I can take a look.